docs/install-mlrun-ce/kubernetes-install.md

```diff
@@ -0,0 +1,262 @@
+(install-on-kubernetes)=
+# Install MLRun CE on Kubernetes
+
+These instructions install the community edition (CE) on your Kubernetes cluster. This procedure installs an EKS cluster, an EBS volume, an S3 bucket, load balancing, etc. When you complete this procedure, you'll have the Community Edition of MLRun running on your EKS cluster.
+
+```{admonition} Note
+These instructions install the community edition {{ ceversion }}, which currently includes the features in MLRun {{ version }}.</br>
+```
+
+**In this section**
+- [Prerequisites](#prerequisites)
+- [Installing the chart](#installing-the-chart)
+- [Usage](#usage)
+- [Start working](#start-working)
+- [Configuring the remote environment](#configuring-the-remote-environment)
+- [Uninstalling the chart](#uninstalling-the-chart)
+- [Upgrading the chart](#upgrading-the-chart)
+
+## Prerequisites
+
+- Access to a Kubernetes cluster, version >=1.32. To install MLRun on your cluster, you must have administrator permissions. 
+For local installation on Windows or Mac, [Docker Desktop](https://www.docker.com/products/docker-desktop) is recommended. 
+- The Kubernetes command-line tool (kubectl) compatible with your Kubernetes cluster is installed. Refer to the [kubectl installation 
+instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for more information.
+- Helm version >=3.16 CLI is installed. Refer to the [Helm installation instructions](https://helm.sh/docs/intro/install/) for more information.
+- An accessible Docker Registry (such as [Docker Hub](https://hub.docker.com)). The Registry's URL and credentials are consumed by the applications via a pre-created secret. If using Docker Hub, the Registry server is `https://registry.hub.docker.com/`. See the [Docker ID documentation](https://docs.docker.com/docker-id/) for details about creating a user with login that you will configure in the secret.
```

Comment:
The note about using docker-hub is repeated later in the detailed installation. Should this be here as well?